osbuild/util/jsoncomm.py

```diff
@@ -128,6 +128,28 @@ def blocking(self, value: bool):
         """Set the blocking mode of the socket."""
         self._socket.setblocking(value)
 
+    def accept(self) -> "Socket":
+        """Accept a new connection on the socket.
+
+        See python's `socket.accept` for more information.
+        """
+        # Since new connection requests, i.e. clients connecting, are
+        # directly put on the receive queue of the listener socket,
+        # accept here will always return a socket and never block,
+        # even if the client meanwhile disconnected
+        conn, _ = self._socket.accept()
+        return Socket(conn, None)
+
+    def listen(self, backlog: Optional[int] = None):
+        """Enable accepting of incoming connections.
+
+        See python's `socket.listen` for details.
+        """
+
+        # `Socket.listen` accepts an `int` or no argument, but not `None`
```

Comment:
FWIW, it is set to `10` by default. Since this is completely unaccounted, non-quota'ed and unpredictable, I would just pass `2^16`. The net-core will truncate it to `sysctl.somaxconn`, which is `SOMAXCONN = 4096` by default. So if we pass `2^16`, we get the system-wide maximum, and we still have the same guarantees.

Reply:
Done